### PR TITLE
DEV-7921 Image replace operation remains enabled when document is out of sync

### PR DESCRIPTION
**Pull request by albertosml**

### CODE REVIEW

File: packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import documentsManager from 'fontoxml-documents/src/documentsManager.js';
+import addAction from 'fontoxml-operations/src/addAction.js';
+import remoteDocumentStateManager from 'fontoxml-remote-documents/src/remoteDocumentStateManager.js';
+
+export default function install() {
+	addAction(
+		'isOutOfSyncDocument',
+		function isOutOfSyncDocument(stepData) {},
```
Comment (bwrrp) [inline packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js]: The execute callback for this action should also be able to cancel the operation in cases where the operation is executed without first checking its state.

File: packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		function getIsOutOfSyncDocumentState(stepData) {
+			var documentId = documentsManager.getDocumentIdByNodeId(stepData.contextNodeId);
+
+			// If document out of sync, we need to disable the operation
+			const documentStatus = remoteDocumentStateManager.getState(documentId);
```
Comment (bwrrp) [inline packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js]: Won't a similar problem occur when the document becomes read-only for some other reason (e.g., when the lock becomes unavailable)?

Comment (albertosml) [reply packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js:13]: Then, I suggest to disable the operation when the document doesn't have a status, that is, the document is fine.

**Comment by albertosml** [comment]
@bwrrp I am going to close this PR, as we don't need it. I will put you as reviewer in the another PR that I'm going to do.

**Pull request closed without merging**